README.md

```diff
@@ -67,10 +67,10 @@ and have the support needed.
 
 | Initiative         | Champion(s)                    | Link                       |
 |--------------------|--------------------------------|----------------------------|
-| Badges             | [amiller-gh]                   | [nodejs/badges]            |
 | i18n               | [obensource]                   | [nodejs/i18n]              |
 | Mentorship         | [Bamieh] and [dshaw]           | [nodejs/mentorship]        |
 | Node.js Collection | [bnb]                          | [nodejs/nodejs-collection] |
+| Outreach           | [Tiriel]                       | [nodejs/outreach]          |
```

Comment:
Need to reconfirm in light of #465

Reply:
@maedahbatool and I would love to volunteer as champions here on the outreach initiative and grow it as we discussed in the first meeting on Tuesday. Peace! ✌️